tests/test_diagram.py

```diff
@@ -47,12 +47,12 @@ def test_validate_curvestyle(self):
     def test_validate_outformat(self):
```

Comment:
There was an issue with the tests, so I decided to modify the code to support the test, if that wasn't the test intention let me know and I will restore the previous